modules/installation-configuration-parameters.adoc

```diff
@@ -1654,96 +1681,122 @@ In {product-title} 4.13, the following vSphere configuration parameters are depr
 The following table lists each deprecated vSphere configuration parameter:
 
 .Deprecated VMware vSphere cluster parameters
-[cols=".^2,.^3a,.^3a",options="header"]
+[cols=".^2a,.^4a,.^2a",options="header"]
```

Comment:
If all of col 1 is literal, why not:

```suggestion
[cols=".^2l,.^4a,.^2a",options="header"]
```

Reply:
(at least I think that's how it would work! I haven't tried this one myself yet :stuck_out_tongue:)

Reply:
The Yeah, I removed these. Originally , I thought they had to do with keeping table entries flush left, but clearly not the case.